hal/halx86/acpi.cmake

```diff
@@ -1,11 +1,18 @@
+include_directories(include ${REACTOS_SOURCE_DIR}/drivers/bus/acpi/acpica/include)
 
 list(APPEND HAL_ACPI_SOURCE
     acpi/halacpi.c
     acpi/halpnpdd.c
     acpi/busemul.c
+    acpi/madt.c
     legacy/bus/pcibus.c)
 
+if(ARCH STREQUAL "amd64")
+    add_definitions(-DWIN64)
+endif()
+
 add_library(lib_hal_acpi OBJECT ${HAL_ACPI_SOURCE})
+add_pch(lib_hal_acpi ${REACTOS_SOURCE_DIR}/drivers/bus/acpi/acpica/include/acpi.h ${HAL_ACPI_SOURCE})
```

Comment:
I believe this was one of those things where only a few compiled were giving me issues: the ACPI_BIOS_ERROR defined in ACPICA doesn't let me undefine on some of the compiler if it isn't compiled before the actual code

Reply:
Looking at it again, it seems this is the only these days built with a PCH.
Since the ACPI hal is tightly integrated with ACPICA eventually in an ideal world this will be used more eventually. Since it's also the only Hal right now using a PCH I feel like it would make sense to leave it as is.